### PR TITLE
Add tooltip name of tech stack on hover #3

### PR DESCRIPTION
**Pull request by AhmdFarzki**
Hi @mittal-parth, I have fixed this problem. Please check and merge if appropriate. Thank you.

Screenshot (86)

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -657,19 +678,23 @@ export const projects = [
       {
         id: "icon-1",
         icon: SiDjango,
+        name: "Django"
       },
 
       {
         id: "icon-2",
         icon: AiOutlineGoogle,
+        name: "Google"
```
Comment (mittal-parth) [inline src/constants/index.js:687]:
```suggestion
        name: "Google OAuth"
```

**Review by mittal-parth — CHANGES_REQUESTED** [review]
Thanks for making the PR!
- [ ] Please checkout the `package-lock.json` file from the PR.
- [ ] Is it possible to add like an upward arrow to the tooltip, so it looks more like a tooltip?

File: src/components/Projects.jsx [hunk 2]
```diff
@@ -28,14 +28,15 @@ const Project = (props) => {
...
             <div className="flex sm:flex-row">
               {props.stack.map((tech, index) => (
-                <span
+                <div
                   key={tech.id}
                   index={index}
-                  className="text-dimWhite mr-5 text-[20px] hover:text-teal-200"
-                  title="less go"
+                  className="text-dimWhite mr-5 text-[20px] hover:text-teal-200 tooltip"
+                  too
```
[the remark this answers is not in the document]
Comment (AhmdFarzki) [reply src/components/Projects.jsx:35]: No, I'm sorry it's my fault

**Comment by AhmdFarzki** [comment]
Sorry i dont understand about "Please checkout the package-lock.json file from the PR." can your explaint this?

**Comment by mittal-parth** [comment]
> Sorry i dont understand about "Please checkout the package-lock.json file from the PR." can your explaint this?

Never mind. Coming from Ruby on Rails background, I thought it shouldn't be commited like Gemfile-lock. You can ignore that.

**Comment by AhmdFarzki** [comment]
Hi @mittal-parth, I have added arrows in the tooltip, please check again.

**Review by AhmdFarzki — COMMENTED** [review]
I've added upward arrow in the tooltip

**Review by mittal-parth — APPROVED** [review]
Looks great!

**Comment by mittal-parth** [comment]
@AhmdFarzki Merged!
Thanks for contributing. Do give it a star if you feel like it! 💙

**Pull request merged**